docs/modules/data-structures/pages/working-with-external-data.adoc

```diff
@@ -31,15 +31,88 @@ NOTE: The data store that you choose must be a centralized system that is access
 
 NOTE: The `MapStore` interface extends the `MapLoader` interface. Therefore, all methods and configuration parameters of the `MapLoader` interface are also available on the `MapStore` interface.
 
+== Defining External Data Stores (Beta)
+
+You can define external data stores in your `hazelcast.xml/hazelcast.yaml` configuration files or xref:configuration:configuring-programmatically.adoc[programmatically]. You can then reference these data stores in the xref:integrate:jdbc-connector.adoc[JDBC Connector] and in `MapStores` and `MapLoaders`.
```

Comment:
```suggestion
You can define external data stores in your members' configuration files or with the xref:configuration:configuring-programmatically.adoc[Java member API]. You can then reference these data stores in the JDBC connector for the xref:integrate:jdbc-connector.adoc[Pipeline API], SQL mappings, and in MapStores and MapLoaders.
```

Reply:
The name of the files add unnecessary detail that may not be true for the reader (these files can be renamed).

Reply:
> You can then reference these data stores in the JDBC connector and in MapStores and MapLoaders.

How do you do this? Do we have an example or advice for doing so?

Reply:
> > You can then reference these data stores in the JDBC connector and in MapStores and MapLoaders.
> 
> How do you do this? Do we have an example or advice for doing so?

Yes, here: https://github.com/hazelcast/hz-docs/pull/514/files#diff-a5e296d87cffbaa364f8bd2d7e6b55ce89d61dbdb71f879a9b23fba2f0cd83bc